ui-v2/tests/acceptance/components/intention-filter.feature

```diff
@@ -0,0 +1,55 @@
+@setupApplicationTest
+Feature: components / intention filter: Intention Filter
+  In order to find the intention I'm looking for easier
+  As a user
+  I should be able to filter by 'policy' (allow/deny) and freetext search tokens by source and destination
+  Scenario: Filtering [Model]
+    Given 1 datacenter model with the value "dc-1"
+    And 2 [Model] models
```

Comment:
I think there's a couple of reasons why I'm doing this.

1. Ideally I'd like to try and get this functionality in 1 feature file (for all the filters across the app), I don't think I'm going to be able to do this, bit more hoping/dreaming more than anything 😄 
2. Not right now for intentions, but theoretically the filters could be used at multiple URLs (like for https://github.com/hashicorp/consul/blob/28ffc75f64d6572c0cd018f4583b74cca5865c05/ui-v2/tests/acceptance/components/catalog-filter.feature#L56-L61). If/when they are I can just add in another line here.

I suppose a smaller reason to do it like this is just to prevent repeating 'intention' everywhere, plus its easy to make tests for new filters by just duplicating another filter test and tweaking the lines in the `Where` statement. I probably copied this from https://github.com/hashicorp/consul/blob/master/ui-v2/tests/acceptance/components/acl-filter.feature and tweaked it a bit. There would have been more to tweak if I'd not used the `Where` stuff. Of course I'd rather do point 1 above, but not sure how straightforward that would be.